app/_src/gateway/reference/rate-limiting.md

```diff
@@ -47,6 +47,18 @@ Module configuration data, such as sync rate, shared dictionary name, storage po
 ### Public Functions
 The following public functions are provided by this library:
 
+`ratelimiting.new_instance`
```

Comment:
```suggestion

{% if_version gte:3.7.x %}
`ratelimiting.new_instance`
```
Since this library was updated in 3.7.

Reply:
Actually we have also backported this to 3.4/3.5/3.6. Should we use `{% if_version gte:3.4.x %}` instead?